monitor/verifier.go

```diff
@@ -37,19 +38,23 @@ func (m *EthMonitor) verifyActiveChanges() {
 			continue
 		}
 
-		if m.verifyChange(change) {
+		verified, approve := m.verifyChange(change)
```

Comment:
what is the reason making this change?

Reply:
no retry verifying if values not changed (e.g., already updated by duplicated sync proposal)